build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -157,6 +157,13 @@
     option: autostart
     value: "false"
 
+- name: Disable grafana
+  ini_file:
+    path: /etc/supervisord.d/grafana.ini
+    section: program:grafana
+    option: autostart
+    value: "false"
```

Comment:
let's enable it after initialization

Reply:
it will slowdown first initialization and we will loose time that we win by running initialization during build time

Reply:
Btw, the block above is not even necessary given the current initial `grafana.ini` [config](https://github.com/percona/pmm/blob/v3/build/ansible/roles/supervisord/files/grafana.ini#L13).

Reply:
Moreover, setting `autostart:true` does not do anything on its own - the service will not restart without supervisord, which either needs to restart or to reread the config.